docker/README.MD

```diff
@@ -0,0 +1,23 @@
+# Docker & Nvidia-docker install
+1. First install [docker-ce](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+2. Install CUDA
+3. Then nvidia-docker https://github.com/NVIDIA/nvidia-docker (don't run the final line)
+
+It's not required to use the image that corresponds to your OS version since docker will just download it. BUT you MUST use the same CUDA version on the host and container.
+
+If you want to use a CUDA version other than 9.1 just change the top line of the Dockerfile. Possible image tags are listed at [dockerhub](https://hub.docker.com/r/nvidia/cuda/).
+
+# Building the image
+From the adept root dir run the following:
+
+`sudo docker build -t adept:ubuntu16.04-cuda9.1-cudnn7 -f /docker/Dockerfile_Ubuntu16.04_cuda9.1_cudnn7 .`
```

Comment:
getting this error:
`unable to prepare context: unable to evaluate symlinks in Dockerfile path: lstat /docker: no such file or directory`